### PR TITLE
Remove axios dependency and fix admin page issue

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupGeneralParams.vue [hunk 1]
```diff
@@ -149,16 +150,19 @@
...
-					}
-				})
-			   	.then(response => {
-			    	this.defaultAppImage.isView = true;
-			    	this.getAppGeneralSettings();
-			  	}).catch(e => {
-			    })
+						'content-Type': 'application/json'
+					},
+					credentials: 'include',
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupGeneralParams.vue]: remove this

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue [hunk 2]
```diff
@@ -318,30 +321,40 @@
...
-                	}
-                	else {
-		        		this.error = this.$t('appCenter.adminSetupForm.error');
-		        	}
-		    	})
+		        return fetch(addEditApplication, {
+		        	headers: {
+		        		'content-Type': 'application/json'
+					},
+					credentials: 'include',
```
Comment (azayati) [inline app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue]: Remove this

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupGeneralParams.vue (hunk 1 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupGeneralParams.vue]: Done!

File: app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue (hunk 2 above)
Comment (SaraBoutej) [reply app-center-webapps/src/main/webapp/vue-apps/adminSetup/components/AdminSetupList.vue]: Done!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**